### ISSUE TITLE
Loading GIF will processing server request

### ISSUE THREAD

**Issue by Chris-Pratt-Clystnet** [rubanraj54/vue-bootstrap4-table#13]
Would it be possible to have a loading icon shown in the table body while requests to the server are being handled.

I request this as I have run into instances where a user has a slow network connection and are getting confused on pages with a data table as all they see is is no data and have no idea that a server request is actually happening.

**Comment by rubanraj54** [comment]
Hi @Chris-Pratt-Clystnet ,

That's a valid request and I've already had this feature in my todo list.
So Probably I will implement this feature in my upcoming release.

Cheers,
Ruby.

**Comment by Chris-Pratt-Clystnet** [comment]
Thanks for the info I will keep a watch out for this update

**Comment by pordonez** [comment]
It would be nice if we could control/bind to the "No results found" message.  Thank you.

**Comment by rubanraj54** [comment]
Hi @pordonez ,

You could use "empty-results" slot to override the "No results found" message.

**Example:**
```vue
<vue-bootstrap4-table :rows="rows"
                    :columns="columns"
                    :config="config"
                    :actions="actions">

    <template slot="empty-results">
        your custom message
    </template>
</vue-bootstrap4-table>
```

Cheers,
Ruby.

**Comment by Chris-Pratt-Clystnet** [comment]
Any news on this feature ?

**Comment by rubanraj54** [comment]
Hey @Chris-Pratt-Clystnet ,

Nowadays I am busy with some other projects but still I spend some time for this.
This weekend I will try to implement this feature.

Cheers,
Ruby.

**Comment by rubanraj54** [comment]
Hey @Chris-Pratt-Clystnet, This feature is implemented and available from version 1.1.11
